ckanext/stcndm/schemas/cube.yaml

```diff
@@ -33,13 +33,15 @@ dataset_fields:
   label:
     en: Content Type
     fr: Type de contenu
-  preset: shortcode_multivalue
+  preset: codeset_select
+  codeset_type: content_type
 
 - field_name: tracking_code
   label:
     en: Display and Tracking
     fr: Affichage et pistage
-  preset: shortcode_multivalue
+  preset: codeset_select
+  codeset_type: tracking
```

Comment:
Just as a personal preference I added an extra field to the schema to explicitly choose the codeset_type (instead of something trickier like chopping the "_code" off the field name)